src/framework/preferences/js/SyllabificationEnactor.js

```diff
@@ -350,63 +357,83 @@ var fluid_3_0_0 = fluid_3_0_0 || {};
     };
 
     /**
+     * Splits a text node at the specified `position` into two text nodes and inserts the specified `toInsert` content
+     * between them.
      *
+     * @param {DomNode} node - The text node to be split by the inserted content
+     * @param {DomNode|DomElement} toInsert - The node/element to insert into the original content from the `node`
+     * @param {Number} position - The position to split `node`. At this point the `node` will be split into two new text
+     *                            nodes and the `toInsert` node placed between them. The position is the 0 based index
+     *                            where the content would be inserted into the original text node. That is, the portion
+     *                            before `position` will be included before the inserted node, and those including and
+     *                            after `position` will be included after the inserted node.
+     *
+     * @return {DomNode} - The newly created text node after executing the split. That is, the second portion of the
+     *                     severed text node.
+     */
+    fluid.prefs.enactor.syllabification.insertIntoTextNode = function (node, toInsert, position) {
+        node = node.splitText(position);
+        node.parentNode.insertBefore(toInsert, node);
+        return node;
+    };
+
+    /**
+     * Insert separators, indicating the hyphenated positions, into a text node. This will inject the specified
+     * markup as the separators and split the original textnode at the hpyhenation points. If there are no places to
+     * inject a hyphenation point, the node will be left unchanged. If the original textnode inclues soft hyphens (i.e.
+     * &shy;) characters, these will be used as a hyphenation point with the original location in the text node
+     * replacded with the `softHyphenPlaceholderMarkup` to allow it to be replaced when syllabification is removed.
```

Comment:
Typos hpyhenation and replacded. You probably want the last "replaced" to be "restored" for a bit of extra clarity